test/CTestList.cmake

```diff
@@ -33,10 +33,13 @@ function(add_test_r TEST_NAME)
     # Copy files to test working directory
     file(COPY ${TEST_FILES} DESTINATION "${CURRENT_TEST_BINARY_DIR}/")
     # Set some default runtime options for all tests in this category
-    set(RUNTIME_OPTIONS "time.max_step=10 amr.plot_file=plt amr.checkpoint_files_output=0 amr.plot_files_output=1 amrex.throw_exception=1 amrex.signal_handling=0")
+    set(RUNTIME_OPTIONS "time.max_step=10 amr.plot_file=plt time.plot_interval=10 time.checkpoint_interval=10 amrex.throw_exception=1 amrex.signal_handling=0")
```

Comment:
did you mean to disable `checkpoint` files here? Previously that was 0 now it is 10.

Reply:
Now that we have a restart test, I can't globally disable checkpoints. (that option didn't work anyway since it was for PeleC and names have changed here) I would like to disable checkpoints for tests though.

Reply:
Actually I realize I need to redesign the restart test because it complicates these runtime options even further.

Reply:
Ok I think I figured it out and will just rely on each test to appropriately set the checkpoint intervals if a restart test depends on it, but still force plot intervals.